github/Consts.py

```diff
@@ -125,3 +125,12 @@
 
 # https://developer.github.com/changes/2018-10-16-deployments-environments-states-and-auto-inactive-updates/
 deploymentStatusEnhancementsPreview = "application/vnd.github.flash-preview+json"
+
+
+DEFAULT_BASE_URL = "https://api.github.com"
```

Comment:
Moved those here since they are used in 2 files now (`MainClass` and  `GithubIntegration`)

Reply:
Comparing this is really difficult. Would have been better to do the refactoring (without changes) in a separate PR and base this PR on that. Then we would see only changes, not moving unchanged code around.